### ISSUE TITLE
yo [generator-name] --help does not list subgenerators for a specific generator

### ISSUE THREAD

**Issue by Hikosei** [yeoman/yo#357]
Hi,
I have read that : `yo --help`give the list of subgenerators, It's working for me. 
But I want the subgenerator for a specific generator not all the generators installed.
Something like `yo my-generator --help`.
Did I missed something ? 

PS: Great job ;)

**Comment by eddiemonge** [comment]
That is already a thing. What generator is it not working for?

**Comment by Hikosei** [comment]
It's on a custom generator, have you a link to the documentation to implement this in my custom generator ?

**Comment by eddiemonge** [comment]
See http://yeoman.io/authoring/index.html

**Comment by Hikosei** [comment]
When i do 
`yo --help`

I see all the subgenerators availables:
```
  angular
    common
    constant
    controller
    decorator
    directive
    factory
    filter
    main
    provider
    route
    service
    value
    view
  karma
```
Then I try for a specific generator : 
` yo angular --help`

And I have this result :
```
Usage:
  yo angular:app [options] [<appname>]

Options:
  -h,   --help                # Print generator's options and usage                   Default: false
        --app-suffix          # Allow a custom suffix to be added to the module name  Default: false
        --appPath             # Allow to choose where to write the files              Default: false
        --coffee              # Generate CoffeeScript instead of JavaScript           Default: false
        --angular:common      # Angular:common to be invoked                          Default:
        --angular:main        # Angular:main to be invoked                            Default:
        --angular:controller  # Angular:controller to be invoked                      Default:

Arguments:
  appname  # Type: String  Required: false
```

**Comment by eddiemonge** [comment]
```
        --angular:common      # Angular:common to be invoked                          Default:
        --angular:main        # Angular:main to be invoked                            Default:
        --angular:controller  # Angular:controller to be invoked                      Default:
```

You have that list. The others aren't there because the generator needs to be updated

**Comment by jedmao** [comment]
@eddiemonge I just installed a fresh version of `generator-angular` and I'm seeing the same issue. I checked the repo to find that [all of the sub-generators are listed in the `package.json` under `files`](https://github.com/yeoman/generator-angular/blob/019c99702fd8b87043b12012da0c6bf59e5b4771/package.json#L16-L34), so I'm not understanding why only `common`, `main` and `controller` are showing when I run `yo angular --help`.

You say the generator needs to be updated. What, exactly, in the generator needs to be updated?
